server/graphql/v2/object/Host.js

```diff
@@ -727,6 +728,28 @@ export const Host = new GraphQLObjectType({
         description: 'Returns whether the host is trusted or not',
         resolve: account => get(account, 'data.isTrustedHost', false),
       },
+      hasDisputedOrders: {
+        type: new GraphQLNonNull(GraphQLBoolean),
+        description: 'Returns whether the host has any disputed orders',
+        async resolve(collective) {
+          const [results] = await sequelize.query(
+            `
+            SELECT o.id FROM "Orders" o
+            JOIN "Transactions" t ON t."OrderId" = o.id
+            WHERE t."HostCollectiveId" = :hostCollectiveId AND o.status = 'DISPUTED'
+            GROUP BY o.id;
+            `,
+            {
+              type: QueryTypes.SELECT,
+              replacements: {
+                hostCollectiveId: collective.id,
+              },
+            },
+          );
```

Comment:
Not mandatory, but for simple queries, I think a sequelize query would be more consistent here.

Reply:
Agreed, I couldn't quite figure out how to make this call using a sequelize query so I just wrote it out in SQL.